cmd.sh

```diff
@@ -67,6 +67,58 @@ micromamba activate upload-nightly-action
 # trim trailing slashes from $INPUT_ARTIFACTS_PATH
 INPUT_ARTIFACTS_PATH="${INPUT_ARTIFACTS_PATH%/}"
 
+# TODO: First get a list of just the package names so
+# this doesn't have to get run for every wheel
```

Comment:
Need to do this first before this can be ready for review so that this loop isn't run for _every_ wheel.